lerobot/common/policies/vqbet/configuration_vqbet.py

```diff
@@ -0,0 +1,149 @@
+from dataclasses import dataclass, field
+
+
+@dataclass
+class VQBeTConfig:
+    """Configuration class for VQ-BeT.
+
+    Defaults are configured for training with PushT providing proprioceptive and single camera observations.
+
+    The parameters you will most likely need to change are the ones which depend on the environment / sensors.
+    Those are: `input_shapes` and `output_shapes`.
+
+    Notes on the inputs and outputs:
+        - "observation.state" is required as an input key.
+        - At least one key starting with "observation.image is required as an input.
+        - If there are multiple keys beginning with "observation.image" they are treated as multiple camera
+          views. Right now we only support all images having the same shape.
+        - "action" is required as an output key.
+
+    Args:
+        n_obs_steps: Number of environment steps worth of observations to pass to the policy (takes the
+            current step and additional steps going back).
+        n_action_pred_token: Number of future tokens that VQ-BeT predicts.
```

Comment:
If I understand correctly, this is not just future tokens but also includes the "current" token. Right?

Reply:
Thank you, I changed this comment:
`n_action_pred_token: Total number of current token and future tokens that VQ-BeT predicts.`